auth/src/main/scala/org/scalatra/auth/ScentryStrategy.scala

```diff
@@ -7,7 +7,7 @@ trait ScentryStrategy[UserType <: AnyRef] {
 
   protected def app: ScalatraBase
 
-  def name: String = "NameMe"
+  def name: String = this.getClass.getName
 
   def registerWith(registrar: Scentry[UserType]): Unit = {
     if (name == "NameMe") throwOverrideException
```

Comment:
Wouldn't the logic to check the default NameMe and issue an exception be unnecessary?

Reply:
As throwOverrideException says
```This method is used when configuring strategies through web.xml```

I'm wiring up my strategies in code, not config so this doesn't get called.